packages/ui5-tooling-transpile/lib/middleware.js

```diff
@@ -23,7 +23,10 @@ const { createBabelConfig, normalizeLineFeeds } = require("./util");
 module.exports = async function ({ resources, options /*, middlewareUtil */ }) {
 	const config = options?.configuration || {};
 	config.includes = config.includes || config.includePatterns;
-	config.excludes = config.excludes || config.excludePatterns;
+
+	// never transpile these default exclusion types
+	const defaultExcludes = [".png", "jped", ".jpg"];
+	config.excludes = defaultExcludes.concat(config.excludes || config.excludePatterns);
```

Comment:
if both `config.excludes` and `config.excludePatterns` are both undefined the concat will add `undefined` - please add another or to add an empty array:
`config.excludes || config.excludePatterns || []`

Reply:
BTW: there is also a typo: `"jped"` should be  `".jpeg"`, right?